.pre-commit-config.yaml

```diff
@@ -0,0 +1,29 @@
+exclude: '^docs/|/migrations/|devcontainer.json'
+default_stages: [commit]
+
+default_language_version:
+  python: python3.11
```

Comment:
python 3.10

https://docs.dify.ai/getting-started/install-self-hosted/local-source-code#installation-of-the-basic-environment